### DISCUSSION TITLE
Will Appium flutter driver supports only the debug builds or It will support the release build for android and app store builds for iOS?

### DISCUSSION THREAD

**Discussion by VenkateshSeenivasan** [appium/appium#18563] [Q&A] [answered]
Will Appium flutter driver supports only the debug builds or It will support the release build for android and app store builds for iOS? 
Do we need to anything else to make it works in release and app store builds at the time of application development ?

**Comment by jlipps** [comment]
The Appium team doesn't maintain the flutter driver. You might be better off also asking in the forums or the flutter driver repo.

**Comment by VenkateshSeenivasan** [comment]
I can see the reference related to the Appium flutter driver. Could you please take a look at it @jlipps 
https://github.com/appium-userland/appium-flutter-driver

**Comment by KazuCocoa** [comment, marked as answer]
For a release build, simply you should use UIA2/XCUITest since a release build in Flutter does not expose Dart VM. It means the build cannot provide Flutter-specific functionality. (It is Flutter's spec)

**Reply by subhanmansuri** [reply]
@KazuCocoa 

we are using the debug build with enableFlutterDriverExtension() then also getting the error for 

cannot read propety 'match' undefined 

Mac OS system 

Android automation testing script 

FlutterApplicaiton

Attaching the log

Waiting until the server is closed
[HTTP] Received server close event
[Appium] Welcome to Appium v1.22.3
[Appium] Non-default server args:
[Appium]   address: 127.0.0.1
[Appium]   port: 4729
[Appium]   relaxedSecurityEnabled: true
[Appium]   allowInsecure: {
[Appium]   }
[Appium]   denyInsecure: {
[Appium]   }
[Appium] Appium REST http interface listener started on 127.0.0.1:4729
[HTTP] --> GET /wd/hub/status
[HTTP] {}
[GENERIC] Calling AppiumDriver.getStatus() with args: []
[GENERIC] Responding to client with driver.getStatus() result: {"build":{"version":"1.22.3"}}
[HTTP] <-- GET /wd/hub/status 200 7 ms - 68
[HTTP] 
[HTTP] Request idempotency key: 87eec847-33b8-47c0-b615-f005ea4cd90e
[HTTP] --> POST /wd/hub/session
[HTTP] {"capabilities":{"firstMatch":[{}],"alwaysMatch":{"appium:app":"/Users/administrator/halsaflutter/src/main/resources/app/app-debug.apk","appium:automationName":"flutter","appium:dartObservatoryPollTimeout":200,"appium:fullReset":false,"appium:newCommandTimeout":300,"appium:noReset":true,"appium:platformVersion":"12.0","appium:udid":"3f0d86e8","platformName":"android"}}}
[W3C] Calling AppiumDriver.createSession() with args: [null,null,{"firstMatch":[{}],"alwaysMatch":{"appium:app":"/Users/administrator/halsaflutter/src/main/resources/app/app-debug.apk","appium:automationName":"flutter","appium:dartObservatoryPollTimeout":200,"appium:fullReset":false,"appium:newCommandTimeout":300,"appium:noReset":true,"appium:platformVersion":"12.0","appium:udid":"3f0d86e8","platformName":"android"}}]
[BaseDriver] Event 'newSessionRequested' logged at 1726570534619 (16:25:34 GMT+0530 (India Standard Time))
[Appium] Appium v1.22.3 creating new FlutterDriver (v0.0.32) session
[Appium] Applying relaxed security to 'FlutterDriver' as per server command line argument. All insecure features will be enabled unless explicitly disabled by --deny-insecure
[BaseDriver] Creating session with W3C capabilities: {
[BaseDriver]   "alwaysMatch": {
[BaseDriver]     "platformName": "android",
[BaseDriver]     "appium:app": "/Users/administrator/halsaflutter/src/main/resources/app/app-debug.apk",
[BaseDriver]     "appium:automationName": "flutter",
[BaseDriver]     "appium:dartObservatoryPollTimeout": 200,
[BaseDriver]     "appium:fullReset": false,
[BaseDriver]     "appium:newCommandTimeout": 300,
[BaseDriver]     "appium:noReset": true,
[BaseDriver]     "appium:platformVersion": "12.0",
[BaseDriver]     "appium:udid": "3f0d86e8"
[BaseDriver]   },
[BaseDriver]   "firstMatch": [
[BaseDriver]     {}
[BaseDriver]   ]
[BaseDriver] }
[BaseDriver] The following capabilities were provided, but are not recognized by Appium:
[BaseDriver]   app
[BaseDriver]   dartObservatoryPollTimeout
[BaseDriver] Session created with session id: eff6fb7d-75de-4f3a-9689-d1c9e3718a7f
[FlutterDriver] Starting an Android proxy session
[BaseDriver] Creating session with W3C capabilities: {
[BaseDriver]   "alwaysMatch": {
[BaseDriver]     "platformName": "android",
[BaseDriver]     "appium:app": "/Users/administrator/halsaflutter/src/main/resources/app/app-debug.apk",
[BaseDriver]     "appium:automationName": "flutter",
[BaseDriver]     "appium:dartObservatoryPollTimeout": 200,
[BaseDriver]     "appium:fullReset": false,
[BaseDriver]     "appium:newCommandTimeout": 300,
[BaseDriver]     "appium:noReset": true,
[BaseDriver]     "appium:platformVersion": "12.0",
[BaseDriver]     "appium:udid": "3f0d86e8"
[BaseDriver]   },
[BaseDriver]   "firstMatch": [
[BaseDriver]     {}
[BaseDriver]   ]
[BaseDriver] }
[BaseDriver] The following capabilities were provided, but are not recognized by Appium:
[BaseDriver]   dartObservatoryPollTimeout
[BaseDriver] Session created with session id: 98a57c5f-2ce3-47a4-a130-32bedfc02c2a
[BaseDriver] Using local app '/Users/administrator/halsaflutter/src/main/resources/app/app-debug.apk'
[UiAutomator2] Checking whether app is actually present
[ADB] Using 'adb' from '/Users/administrator/Library/Android/sdk/platform-tools/adb'
[ADB] Running '/Users/administrator/Library/Android/sdk/platform-tools/adb -P 5037 start-server'
[AndroidDriver] Retrieving device list
[ADB] Trying to find a connected android device
[ADB] Getting connected devices
[ADB] Connected devices: [{"udid":"3f0d86e8","state":"device"}]
[AndroidDriver] Using device: 3f0d86e8
[ADB] Using 'adb' from '/Users/administrator/Library/Android/sdk/platform-tools/adb'
[ADB] Running '/Users/administrator/Library/Android/sdk/platform-tools/adb -P 5037 start-server'
[ADB] Setting device id to 3f0d86e8
[ADB] Running '/Users/administrator/Library/Android/sdk/platform-tools/adb -P 5037 -s 3f0d86e8 shell getprop ro.build.version.sdk'
[ADB] Current device property 'ro.build.version.sdk': 31
[ADB] Getting device platform version
[ADB] Running '/Users/administrator/Library/Android/sdk/platform-tools/adb -P 5037 -s 3f0d86e8 shell getprop ro.build.version.release'
[ADB] Current device property 'ro.build.version.release': 12
[ADB] Device API level: 31
[UiAutomator2] Relaxing hidden api policy
[ADB] Running '/Users/administrator/Library/Android/sdk/platform-tools/adb -P 5037 -s 3f0d86e8 shell 'settings put global hidden_api_policy_pre_p_apps 1;settings put global hidden_api_policy_p_apps 1;settings put global hidden_api_policy 1''
[AndroidDriver] Parsing package and activity from app manifest
[ADB] Package name: 'com.my.med'
[ADB] Main activity name: 'com.my.med.MainActivity'
[AndroidDriver] Parsed package and activity are: com.my.med/com.my.med.MainActivity
[ADB] Running '/Users/administrator/Library/Android/sdk/platform-tools/adb -P 5037 -s 3f0d86e8 wait-for-device'
[ADB] Running '/Users/administrator/Library/Android/sdk/platform-tools/adb -P 5037 -s 3f0d86e8 shell echo ping'
[AndroidDriver] Pushing settings apk to device...
[ADB] Getting install status for io.appium.settings
[ADB] Running '/Users/administrator/Library/Android/sdk/platform-tools/adb -P 5037 -s 3f0d86e8 shell dumpsys package io.appium.settings'
[ADB] 'io.appium.settings' is installed
[ADB] Getting package info for 'io.appium.settings'
[ADB] Running '/Users/administrator/Library/Android/sdk/platform-tools/adb -P 5037 -s 3f0d86e8 shell dumpsys package io.appium.settings'
[ADB] The version code of the installed 'io.appium.settings' is greater than the application version code (72 > 30)
[ADB] There is no need to downgrade 'io.appium.settings'
[ADB] Getting IDs of all 'io.appium.settings' processes
[ADB] Running '/Users/administrator/Library/Android/sdk/platform-tools/adb -P 5037 -s 3f0d86e8 shell 'pgrep --help; echo $?''
[ADB] Running '/Users/administrator/Library/Android/sdk/platform-tools/adb -P 5037 -s 3f0d86e8 shell pgrep -f \(\[\[:blank:\]\]\|\^\)io\.appium\.settings\(\[\[:blank:\]\]\|\$\)'
[AndroidDriver] io.appium.settings is already running. There is no need to reset its permissions.
[ADB] Running '/Users/administrator/Library/Android/sdk/platform-tools/adb -P 5037 -s 3f0d86e8 shell appops set io.appium.settings android:mock_location allow'
[Logcat] Starting logs capture with command: /Users/administrator/Library/Android/sdk/platform-tools/adb -P 5037 -s 3f0d86e8 logcat -v threadtime
[UiAutomator2] Forwarding UiAutomator2 Server port 6790 to local port 8203
[ADB] Forwarding system: 8203 to device: 6790
[ADB] Running '/Users/administrator/Library/Android/sdk/platform-tools/adb -P 5037 -s 3f0d86e8 forward tcp:8203 tcp:6790'
[ADB] Getting install status for io.appium.uiautomator2.server
[ADB] Running '/Users/administrator/Library/Android/sdk/platform-tools/adb -P 5037 -s 3f0d86e8 shell dumpsys package io.appium.uiautomator2.server'
[ADB] 'io.appium.uiautomator2.server' is installed
[ADB] Getting package info for 'io.appium.uiautomator2.server'
[ADB] Running '/Users/administrator/Library/Android/sdk/platform-tools/adb -P 5037 -s 3f0d86e8 shell dumpsys package io.appium.uiautomator2.server'
[ADB] The version name of the installed 'io.appium.uiautomator2.server' is greater or equal to the application version name ('4.27.0' >= '4.27.0')
[UiAutomator2] io.appium.uiautomator2.server installation state: sameVersionInstalled
[ADB] Checking app cert for /Applications/Appium Server GUI.app/Contents/Resources/app/node_modules/appium/node_modules/appium-uiautomator2-server/apks/appium-uiautomator2-server-v4.27.0.apk
[ADB] Using the previously cached signature entry for 'appium-uiautomator2-server-v4.27.0.apk'
[ADB] sha256 hash did match for 'appium-uiautomator2-server-v4.27.0.apk'
[ADB] Getting install status for io.appium.uiautomator2.server.test
[ADB] Running '/Users/administrator/Library/Android/sdk/platform-tools/adb -P 5037 -s 3f0d86e8 shell dumpsys package io.appium.uiautomator2.server.test'
[ADB] 'io.appium.uiautomator2.server.test' is installed
[ADB] Checking app cert for /Applications/Appium Server GUI.app/Contents/Resources/app/node_modules/appium/node_modules/appium-uiautomator2-server/apks/appium-uiautomator2-server-debug-androidTest.apk
[ADB] Using the previously cached signature entry for 'appium-uiautomator2-server-debug-androidTest.apk'
[ADB] sha256 hash did match for 'appium-uiautomator2-server-debug-androidTest.apk'
[UiAutomator2] Server packages are not going to be (re)installed
[UiAutomator2] Waiting up to 30000ms for services to be available
[ADB] Running '/Users/administrator/Library/Android/sdk/platform-tools/adb -P 5037 -s 3f0d86e8 shell pm list instrumentation'
[UiAutomator2] Instrumentation target 'io.appium.uiautomator2.server.test/androidx.test.runner.AndroidJUnitRunner' is available
[ADB] Adding packages ["io.appium.settings","io.appium.uiautomator2.server","io.appium.uiautomator2.server.test"] to Doze whitelist
[ADB] Got the following command chunks to execute: [["dumpsys","deviceidle","whitelist","+io.appium.settings",";","dumpsys","deviceidle","whitelist","+io.appium.uiautomator2.server",";","dumpsys","deviceidle","whitelist","+io.appium.uiautomator2.server.test",";"]]
[ADB] Running '/Users/administrator/Library/Android/sdk/platform-tools/adb -P 5037 -s 3f0d86e8 shell dumpsys deviceidle whitelist +io.appium.settings ; dumpsys deviceidle whitelist +io.appium.uiautomator2.server ; dumpsys deviceidle whitelist +io.appium.uiautomator2.server.test ;'
[ADB] Checking app cert for /Users/administrator/halsaflutter/src/main/resources/app/app-debug.apk
[ADB] Using the previously cached signature entry for 'app-debug.apk'
[ADB] Getting install status for com.my.med
[ADB] Running '/Users/administrator/Library/Android/sdk/platform-tools/adb -P 5037 -s 3f0d86e8 shell dumpsys package com.my.med'
[ADB] 'com.my.med' is installed
[ADB] Getting package info for 'com.my.med'
[ADB] Running '/Users/administrator/Library/Android/sdk/platform-tools/adb -P 5037 -s 3f0d86e8 shell dumpsys package com.my.med'
[ADB] The version name of the installed 'com.my.med' is greater or equal to the application version name ('2.0.9' >= '2.0.9')
[ADB] There is no need to install/upgrade '/Users/administrator/halsaflutter/src/main/resources/app/app-debug.apk'
[UiAutomator2] Performing shallow cleanup of automation leftovers
[UiAutomator2] No obsolete sessions have been detected (socket hang up)
[ADB] Running '/Users/administrator/Library/Android/sdk/platform-tools/adb -P 5037 -s 3f0d86e8 shell am force-stop io.appium.uiautomator2.server.test'
[UiAutomator2] Starting UIAutomator2 server 4.27.0
[UiAutomator2] Using UIAutomator2 server from '/Applications/Appium Server GUI.app/Contents/Resources/app/node_modules/appium/node_modules/appium-uiautomator2-server/apks/appium-uiautomator2-server-v4.27.0.apk' and test from '/Applications/Appium Server GUI.app/Contents/Resources/app/node_modules/appium/node_modules/appium-uiautomator2-server/apks/appium-uiautomator2-server-debug-androidTest.apk'
[UiAutomator2] Waiting up to 30000ms for UiAutomator2 to be online...
[ADB] Creating ADB subprocess with args: ["-P",5037,"-s","3f0d86e8","shell","am","instrument","-w","-e","disableAnalytics",true,"io.appium.uiautomator2.server.test/androidx.test.runner.AndroidJUnitRunner"]
[WD Proxy] Matched '/status' to command name 'getStatus'
[WD Proxy] Proxying [GET /status] to [GET http://127.0.0.1:8203/wd/hub/status] with no body
[WD Proxy] socket hang up
[WD Proxy] Matched '/status' to command name 'getStatus'
[WD Proxy] Proxying [GET /status] to [GET http://127.0.0.1:8203/wd/hub/status] with no body
[WD Proxy] socket hang up
[WD Proxy] Matched '/status' to command name 'getStatus'
[WD Proxy] Proxying [GET /status] to [GET http://127.0.0.1:8203/wd/hub/status] with no body
[WD Proxy] socket hang up
[Instrumentation] io.appium.uiautomator2.server.test.AppiumUiAutomator2Server:
[WD Proxy] Matched '/status' to command name 'getStatus'
[WD Proxy] Proxying [GET /status] to [GET http://127.0.0.1:8203/wd/hub/status] with no body
[WD Proxy] socket hang up
[WD Proxy] Matched '/status' to command name 'getStatus'
[WD Proxy] Proxying [GET /status] to [GET http://127.0.0.1:8203/wd/hub/status] with no body
[WD Proxy] Got response with status 200: {"sessionId":"None","value":{"message":"UiAutomator2 Server is ready to accept commands","ready":true}}
[UiAutomator2] The initialization of the instrumentation process took 4133ms
[WD Proxy] Matched '/session' to command name 'createSession'
[WD Proxy] Proxying [POST /session] to [POST http://127.0.0.1:8203/wd/hub/session] with body: {"capabilities":{"firstMatch":[{"platform":"LINUX","webStorageEnabled":false,"takesScreenshot":true,"javascriptEnabled":true,"databaseEnabled":false,"networkConnectionEnabled":true,"locationContextEnabled":false,"warnings":{},"desired":{"platformName":"android","app":"/Users/administrator/halsaflutter/src/main/resources/app/app-debug.apk","automationName":"flutter","dartObservatoryPollTimeout":200,"fullReset":false,"newCommandTimeout":300,"noReset":true,"platformVersion":"12.0","udid":"3f0d86e8"},"platformName":"android","app":"/Users/administrator/halsaflutter/src/main/resources/app/app-debug.apk","automationName":"flutter","dartObservatoryPollTimeout":200,"fullReset":false,"newCommandTimeout":300,"noReset":true,"platformVersion":"12.0","udid":"3f0d86e8","deviceName":"3f0d86e8","deviceUDID":"3f0d86e8","appPackage":"com.my.med"}],"alwaysMatch":{}}}
[WD Proxy] Got response with status 200: {"sessionId":"9c3cd985-f8db-4924-a4de-ff08d15182e9","value":{"capabilities":{"firstMatch":[{"platform":"LINUX","webStorageEnabled":false,"takesScreenshot":true,"javascriptEnabled":true,"databaseEnabled":false,"networkConnectionEnabled":true,"locationContextEnabled":false,"warnings":{},"desired":{"platformName":"android","app":"/Users/administrator/halsaflutter/src/main/resources/app/app-debug.apk","automationName":"flutter","dartObservatoryPollTimeout":200,"fullReset":false,"newCommandTimeout":300,"noReset":true,"platformVersion":"12.0","udid":"3f0d86e8"},"platformName":"android","app":"/Users/administrator/halsaflutter/src/main/resources/app/app-debug.apk","automationName":"flutter","dartObservatoryPollTimeout":200,"fullReset":false,"newCommandTimeout":300,"noReset":true,"platformVersion":"12.0","udid":"3f0d86e8","deviceName":"3f0d86e8","deviceUDID":"3f0d86e8","appPackage":"com.my.med"}],"alwaysMatch":{}},"sessionId":"9c3cd985-f8db-4924-a4de-ff08d15182e9"}}
[WD Proxy] Determined the downstream protocol as 'W3C'
[WD Proxy] Proxying [GET /appium/device/info] to [GET http://127.0.0.1:8203/wd/hub/session/9c3cd985-f8db-4924-a4de-ff08d15182e9/appium/device/info] with no body
[WD Proxy] Got response with status 200: {"sessionId":"9c3cd985-f8db-4924-a4de-ff08d15182e9","value":{"androidId":"b1035fadc59c2fcb","apiVersion":"31","bluetooth":{"state":"OFF"},"brand":"Redmi","carrierName":"","displayDensity":440,"locale":"en_IN","manufacturer":"Xiaomi","model":"Redmi Note 9 Pro","networks":[{"capabilities":{"SSID":null,"linkDownBandwidthKbps":51977,"linkUpstreamBandwidthKbps":41354,"networkCapabilities":"NET_CAPABILITY_NOT_METERED,NET_CAPABILITY_INTERNET,NET_CAPABILITY_NOT_RESTRICTED,NET_CAPABILITY_TRUSTED,NET_CAPABILITY_NOT_VPN,NET_CAPABILITY_VALIDATED,NET_CAPABILITY_NOT_ROAMING,NET_CAPABILITY_FOREGROUND,NET_CAPABILITY_NOT_CONGESTED,NET_CAPABILITY_NOT_SUSPENDED","signalStrength":-42,"transportTypes":"TRANSPORT_WIFI"},"detailedState":"CONNECTED","extraInfo":"","isAvailable":true,"isConnected":true,"isFailover":false,"isRoaming":false,"state":"CONNECTED","subtype":-1,"subtypeName":"","type":1,"typeName":"WIFI"}],"platformVersion":"12","realDisplaySize":"1080x2400","timeZone":"Europe/Stockholm"}}
[ADB] Running '/Users/administrator/Library/Android/sdk/platform-tools/adb -P 5037 -s 3f0d86e8 shell dumpsys window'
[AndroidDriver] Screen already unlocked, doing nothing
[UiAutomator2] Starting 'com.my.med/com.my.med.MainActivity and waiting for 'com.my.med/com.my.med.MainActivity'
[ADB] Running '/Users/administrator/Library/Android/sdk/platform-tools/adb -P 5037 -s 3f0d86e8 shell am start -W -n com.my.med/com.my.med.MainActivity -S'
[ADB] Waiting up to 20000ms for activity matching pkg: 'com.my.med' and activity: 'com.my.med.MainActivity' to be focused
[ADB] Possible activities, to be checked: 'com.my.med.MainActivity', 'com.my.med.com.my.med.MainActivity'
[ADB] Getting focused package and activity
[ADB] Running '/Users/administrator/Library/Android/sdk/platform-tools/adb -P 5037 -s 3f0d86e8 shell dumpsys window displays'
[ADB] Found package: 'com.my.med' and fully qualified activity name : 'com.my.med.MainActivity'
[WD Proxy] Proxying [GET /appium/device/pixel_ratio] to [GET http://127.0.0.1:8203/wd/hub/session/9c3cd985-f8db-4924-a4de-ff08d15182e9/appium/device/pixel_ratio] with no body
[WD Proxy] Got response with status 200: {"sessionId":"9c3cd985-f8db-4924-a4de-ff08d15182e9","value":2.75}
[WD Proxy] Matched '/appium/device/system_bars' to command name 'getSystemBars'
[WD Proxy] Proxying [GET /appium/device/system_bars] to [GET http://127.0.0.1:8203/wd/hub/session/9c3cd985-f8db-4924-a4de-ff08d15182e9/appium/device/system_bars] with no body
[WD Proxy] Got response with status 200: {"sessionId":"9c3cd985-f8db-4924-a4de-ff08d15182e9","value":{"statusBar":102}}
[WD Proxy] Matched '/window/current/size' to command name 'getWindowSize'
[WD Proxy] Proxying [GET /window/current/size] to [GET http://127.0.0.1:8203/wd/hub/session/9c3cd985-f8db-4924-a4de-ff08d15182e9/window/current/size] with no body
[WD Proxy] Got response with status 200: {"sessionId":"9c3cd985-f8db-4924-a4de-ff08d15182e9","value":{"height":2168,"width":1080}}
[UiAutomator2] Deleting UiAutomator2 session
[UiAutomator2] Deleting UiAutomator2 server session
[WD Proxy] Matched '/' to command name 'deleteSession'
[WD Proxy] Proxying [DELETE /] to [DELETE http://127.0.0.1:8203/wd/hub/session/9c3cd985-f8db-4924-a4de-ff08d15182e9] with no body
[WD Proxy] Got response with status 200: {"sessionId":"9c3cd985-f8db-4924-a4de-ff08d15182e9","value":null}
[ADB] Running '/Users/administrator/Library/Android/sdk/platform-tools/adb -P 5037 -s 3f0d86e8 shell am force-stop com.my.med'
[Logcat] Stopping logcat capture
[ADB] Removing forwarded port socket connection: 8203 
[ADB] Running '/Users/administrator/Library/Android/sdk/platform-tools/adb -P 5037 -s 3f0d86e8 forward --remove tcp:8203'
[UiAutomator2] Restoring hidden api policy to the device default configuration
[ADB] Running '/Users/administrator/Library/Android/sdk/platform-tools/adb -P 5037 -s 3f0d86e8 shell 'settings delete global hidden_api_policy_pre_p_apps;settings delete global hidden_api_policy_p_apps;settings delete global hidden_api_policy''
[Instrumentation] .
[Instrumentation] Time: 6.145
[Instrumentation] 
[Instrumentation] OK (1 test)
[FlutterDriver] Deleting Flutter Driver session
[BaseDriver] Event 'newSessionStarted' logged at 1726570547726 (16:25:47 GMT+0530 (India Standard Time))
[W3C] Encountered internal error running command: TypeError: Cannot read property 'match' of undefined
[W3C]     at Object.exports.processLogToGetobservatory (/Applications/Appium Server GUI.app/Contents/Resources/app/node_modules/appium/node_modules/appium-flutter-driver/build/driver/lib/sessions/observatory.js:118:9)
[W3C]     at Object.exports.getObservatoryWsUri (/Applications/Appium Server GUI.app/Contents/Resources/app/node_modules/appium/node_modules/appium-flutter-driver/lib/sessions/android.ts:39:21)
[W3C]     at Object.exports.startAndroidSession (/Applications/Appium Server GUI.app/Contents/Resources/app/node_modules/appium/node_modules/appium-flutter-driver/lib/sessions/android.ts:26:30)
[W3C]     at runMicrotasks ()
[W3C]     at processTicksAndRejections (internal/process/task_queues.js:97:5)
[W3C]     at FlutterDriver.exports.createSession (/Applications/Appium Server GUI.app/Contents/Resources/app/node_modules/appium/node_modules/appium-flutter-driver/lib/sessions/session.ts:21:47)
[W3C]     at AppiumDriver.createSession (/Applications/Appium Server GUI.app/Contents/Resources/app/node_modules/appium/lib/appium.js:387:35)
[HTTP] <-- POST /wd/hub/session 500 13114 ms - 667
[HTTP] 
[Instrumentation] The process has exited with code 0

**Reply by KazuCocoa** [reply]
Please use the latest Appium and Appium Flutter driver first. The attached log's version is pretty old.

> [HTTP] Received server close event
> [Appium] Welcome to Appium v1.22.3
> [Appium] Non-default server args:
